### PR TITLE
refined telemetry service procedure

### PR DESCRIPTION
**Pull request by klgill**
https://issues.redhat.com/browse/RHOSPDOC-1994

### CODE REVIEW

File: docs_user/modules/proc_adopting-telemetry-services.adoc [hunk 1]
```diff
@@ -2,28 +2,27 @@
...
+To adopt Telemetry services, you patch an existing `OpenStackControlPlane` custom resource (CR) that has Telemetry services disabled to start the service with the configuration parameters that are provided by the {rhos_prev_long} ({OpenStackShort}) {rhos_prev_ver} environment.
 
-Metrics and logs are considered short-lived data and are not intended to be migrated to the target environment. For information about adopting legacy autoscaling stack templates to the target environment, see xref:adopting-autoscaling_adopt-control-plane[Adopting Autoscaling services].
+If you adopt Telemetry services, the observability solution that is used in the {OpenStackShort} {rhos_prev_ver} environment, Service Telemetry Framework, is removed from the cluster. The new solution is deployed in the {rhos_long} environment, allowing for metrics, and optionally logs, to be retrieved and stored in the new back ends.
```
Comment (klgill) [inline docs_user/modules/proc_adopting-telemetry-services.adoc:7]: Notice that I edited this text further from the last iteration.

File: docs_user/modules/proc_adopting-telemetry-services.adoc [hunk 2]
```diff
@@ -2,28 +2,27 @@
...
 
 .Prerequisites
 
-* Previous Adoption steps completed. MariaDB, the {identity_service_first_ref} and the data plane should be already adopted.
-//kgilliga:Should this procedure be moved after the "Adopting the data plane" chapter?
+* The {OpenStackPreviousInstaller} environment is running (the source cloud).
+* The Single Node OpenShift or OpenShift Local is running in the {OpenShift} cluster.
+* Previous adoption steps are completed.
+* MariaDB, the {identity_service_first_ref}, and the data plane are adopted. For more information, see the following documentation:
+** xref:migrating-databases-to-mariadb-instances_migrating-databases[Migrating databases to MariaDB instances]
```
Comment (klgill) [inline docs_user/modules/proc_adopting-telemetry-services.adoc:17]: One of the prerequisites is that the data plane needs to be adopted.  Should this procedure be moved after the "Adopting the data plane" chapter?

File: docs_user/modules/proc_adopting-telemetry-services.adoc [hunk 3]
```diff
@@ -2,28 +2,27 @@
...
+** xref:adopting-the-identity-service_adopt-control-plane[Adopting the {identity_service}]
+** xref:adopting-data-plane_data-plane[Adopting the data plane]
+//kgilliga: Check the data plane xref after adding ifdef attributes to assemblies.
 
 * Patch the `OpenStackControlPlane` CR to deploy `cluster-observability-operator`:
 +
```
Comment (klgill) [inline docs_user/modules/proc_adopting-telemetry-services.adoc:23]: Would it make sense to move the last 3 steps in the prerequisites to the main procedure?

**Comment by klgill** [comment]
@jlarriba I'm not sure if you're the right person to review this procedure. Please let me know if I should contact someone else.

File: docs_user/modules/proc_adopting-telemetry-services.adoc (hunk 2 above)
Comment (jlarriba) [reply docs_user/modules/proc_adopting-telemetry-services.adoc:17]: No, that sentence is not correct. Please, remove it as all that we need is for the previous adoption steps to be completed.

File: docs_user/modules/proc_adopting-telemetry-services.adoc (hunk 3 above)
Comment (jlarriba) [reply docs_user/modules/proc_adopting-telemetry-services.adoc:23]: I would prefer to get all of this removed, it is enough if the user completed all the previous adoption steps.

**Pull request merged**